issue-001/solution.js

```diff
@@ -1,3 +1,18 @@
 function dataTypeCheck(a) {
     // implement your solution here
+    if  (Number(a) === a) {
```

Comment:
For the 2nd PR, you should not include files for the other issues. Everything should be in isolation.

Make sure you always `git checkout` from the `master` branch as you create the new branches.